Directory.Build.props

```diff
@@ -1,13 +1,13 @@
 <Project>
   <PropertyGroup>
     <LangVersion>12.0</LangVersion>
-    <DotNetTargetFrameworks>net6.0;net7.0;net8.0;net9.0</DotNetTargetFrameworks>
```

Comment:
Please still keep the target framework net9.0. Test project and benchmark project should target to net9.0 too.